generation/3d_ddpm/3d_ddpm_tutorial.ipynb

```diff
@@ -0,0 +1,770 @@
+{
```

Comment:
<p>The description here about the transform doesn't fully align with the code below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1874/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
This is a mismatch with what I was experimenting with previously. I've restored the transforms to their previous state and it the example works now, I had issues getting results previously and images looked better with the new transforms, but I'm going to leave this as it is to match the original example.